frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -70,6 +70,12 @@ const NewMainLayoutContent = styled(Grid)(({ theme }) => ({
         paddingRight: theme.spacing(1),
         margin: 0,
     },
+    [theme.breakpoints.up('xl')]: {
+        maxWidth: '1920px',
+        marginLeft: 'auto',
+        marginRight: 'auto',
+    },
+    minHeight: '94vh',
```

Comment:
<img width="1905" alt="Screenshot 2024-05-27 at 13 16 34" src="https://github.com/Unleash/unleash/assets/1394682/1e4d6efa-a5f5-477e-a59d-1a0e5e5207fe">

Hide footer below the fold so it's never jumping on sidebar navigation expanding